src/scenarios/blazorserverinnerloop/test.py

```diff
@@ -10,7 +10,9 @@
     traits = TestTraits(exename=EXENAME, 
                         startupmetric='InnerLoop',
                         guiapp='false',
-                        innerloopcommandargs='-c "from shutil import copyfile; copyfile(\'src%sStartup.cs\', \'app%sStartup.cs\')"' % (os.sep, os.sep) ,
-                        projext = '.csproj'
+                        innerloopcommandargs='-c "from shutil import copyfile; copyfile(\'src%sProgram.cs\', \'app%sProgram.cs\')"' % (os.sep, os.sep) ,
+                        projext = '.csproj',
+                        processwillexit='false',
+                        measurementdelay='20'
```

Comment:
These were added because we now no longer call exit as part of the execution, but instead use the logging ETW provider to get the application start time, yes?

Reply:
Yes, exit is no longer called in the deprecated startup.cs file so we specify that the process does not exit on it's own.